pandas/tests/frame/methods/test_interpolate.py

```diff
@@ -34,6 +34,10 @@ def test_interp_basic(self):
         expected.loc[5, "B"] = 9
         tm.assert_frame_equal(result, expected)
 
+    def test_interp_empty(self):
+        df = DataFrame()
```

Comment:
can you add the issue number as a comment here

Reply:
can you do this as a code comment, e.g.
```suggestion
        # https://github.com/pandas-dev/pandas/issues/35598
        df = DataFrame()
```